lm_eval/tasks/spanish_bench/spanish_bench.yaml

```diff
@@ -1,6 +1,9 @@
 group: spanish_bench
```

Comment:
@haileyschoelkopf @lintangsutawika Not sure why this is erroring out.
`TypeError: TaskConfig.__init__() got an unexpected keyword argument 'group'`

Reply:
Should now be resolved w/ latest commits!